plugins/CoreAdminHome/Emails/JsTrackingCodeMissingEmail.php

```diff
@@ -71,9 +71,9 @@ public function getIdSite()
     private function setUpEmail()
     {
         $this->setDefaultFromPiwik();
-        $this->addTo($this->emailAddress);
+        $this->addAddress($this->emailAddress);
```

Comment:
any chance we could keep the existing API so we don't need to adjust too many plugins etc? That be great

Reply:
I've added (deprecated) alias methods to the mail class already, so old plugins should still work. Just changed them in core if we decide to remove the old methods at some time...